cmd/kola/options.go

```diff
@@ -15,12 +15,18 @@
 package main
 
 import (
+	"fmt"
 	"github.com/coreos/mantle/kola"
 	"github.com/coreos/mantle/sdk"
 )
 
 var (
-	kolaPlatform string
+	kolaPlatform       string
+	defaultTargetBoard = "amd64-usr"
```

Comment:
Use `sdk.DefaultBoard()` so kola's --board option respects the default just like the build scripts do.